_includes/callout.html

```diff
@@ -1 +1,2 @@
-<div markdown="span" class="bs-callout bs-callout-{{include.type}}">{{include.content}}</div>
+<div markdown="span" class="bs-callout bs-callout-{{include.type}}">{{include.content}}
```

Comment:
Can you move `{{include.content}}` to it's own line.

Reply:
done